replicator/run.py

```diff
@@ -7,6 +7,7 @@
 import requests
 
 
+testing = os.environ.get('DESEC_NS_E2E2', None) is not None
```

Comment:
Easier: `verify = os.environ.get('DESEC_NS_E2E2') is None`, then use `verify` below. (`get()` defaults to `None`.)

Reply:
done